src/main/java/HTTPServer.java

```diff
@@ -0,0 +1,32 @@
+import java.io.*;
```

Comment:
I actually didn't need that library in this class since I moved the logic out to the Server class, so I've removed it here. But I'll definitely look out for more * imports.